vars/perfRegressionParallelPipeline.groovy

```diff
@@ -101,7 +101,7 @@ def call(Map pipelineParams) {
                 }
                 steps {
                     catchError(stageResult: 'FAILURE') {
-                        timeout(time: 1, unit: 'MINUTES') {
+                        timeout(time: 5, unit: 'MINUTES') {
```

Comment:
well, we could align it in some specific point, but the runs that are failing with this timeouts are taking ~2 minutes (that's why i gave it 5 minutes)... but does it really make any difference that this groovy has different timeout than other ones?

Reply:
The stage that gets changed here is common and not related specifically to perf logic.
Sure, it is not strict thing to have single timeouts. Maybe perf CI jobs timeout because of the concrete jenkins builder specifics.
Ok, let's leave it as is.

Reply:
on master we did moved to SCT runners (also on v13)